library_generation/README.md

```diff
@@ -91,14 +91,30 @@ Use `--grpc_version` to specify the value.
 
 Note that if specified, the version should be compatible with gapic-generator-java.
 
+### contains_iam_policy (optional)
+Whether to include `google/iam/v1/iam_policy.proto` into the library.
+The value is either `true` or `false`.
+The default value is `false`.
+
+Use `--contains_iam_policy` to specify the value.
+
+### contains_locations (optional)
+Whether to include `google/cloud/location/locations.proto` into the library.
+The value is either `true` or `false`.
+The default value is `false`.
+
+Use `--contains_locations` to specify the value.
+
```

Comment:
I can see this list growing over time. Would it make sense to just have `additional_protos` parameter?

Reply:
Do you mean `additional_protos` as a string of protos?
For example:
`additional_protos="google/iam/v1/iam_policy.proto google/cloud/location/locations.proto"` or `additional_protos="google/cloud/location/locations.proto"`

Reply:
+1 to make it more generic. It would solve the problem for common shopping [protos](https://github.com/googleapis/googleapis/blob/master/google/shopping/type/types.proto) as well. I'm also curious why `google/longrunning/operations.proto` is not affected by this issue.

Reply:
@blakeli0 post some findings in the [issue](https://github.com/googleapis/sdk-platform-java/issues/1999#issuecomment-1725806816) that answers your question.

Reply:
@blakeli0 @meltsufin I plan to name additional protos that pass to the generator `gapic_protos`, WDYT? 

So far I found `google/cloud/location/locations.proto`, `google/iam/v1/iam_policy.proto` and `google/cloud/common_resources.proto` may go to this list.

Reply:
Sounds good.

Reply:
done